src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -272,12 +271,49 @@ public void fillStepNumScaleBox(String step, int qnNumber) {
         executeScript("$(arguments[0]).change();", stepBox);
     }
     
+    public void fillStepNumScaleBox(double step, int qnNumber) {
+        fillStepNumScaleBox(StringHelper.toDecimalFormatString(step), qnNumber);
+    }
+    
+    public void fillStepNumScaleBoxForNewQuestion(String step) {
+        fillStepNumScaleBox(step, NEW_QUESTION_NUM);
+    }
+    
+    public void fillStepNumScaleBoxForNewQuestion(double step) {
+        fillStepNumScaleBox(step, NEW_QUESTION_NUM);
+    }
+    
     public String getNumScalePossibleValuesString(int qnNumber) {
         String idSuffix = getIdSuffix(qnNumber);
         WebElement possibleValuesSpan = browser.driver.findElement(By.id("numScalePossibleValues" + idSuffix));
         return possibleValuesSpan.getText();
     }
     
+    public String getNumScalePossibleValuesStringForNewQuestion() {
+        return getNumScalePossibleValuesString(-1);
+    }
+    
+    public void fillNumScaleBoxWithRecheck(boolean isMinScaleBox, int scale, int qnNumber, String expected) {
+        int counter = 0;
+        while (counter != 100) {
```

Comment:
Maybe rewrite this with for loop and give counter a better name so that it is easier to read.